test/zdtm/static/cpu-affinity0.c

```diff
@@ -0,0 +1,42 @@
+#include <errno.h>
```

Comment:
The test has to be in a separate patch.

Reply:
Thank you for your reply.
Do I need to commit in two parts, submit the code patch first, and then submit the test patch when it is received?